<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_gestion_usuarios</class>
 <widget class="QWidget" name="admin_gestion_usuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>291</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Encabezados">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="lblFecha">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(133, 133, 133);
</string>
      </property>
      <property name="text">
       <string>Fecha Hoy</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="sideBar">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>0</y>
     <width>300</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 247, 244);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1021</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 238, 238);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>471</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="botonesMenu">
    <item>
     <widget class="QPushButton" name="btnTodo">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	border: none;
	color: #A6A9A5;
}

:hover{
	color: #585858;
}
</string>
      </property>
      <property name="text">
       <string>Todo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMeseros">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	border: none;
	color: #A6A9A5;
}

:hover{
	color: #585858;
}
</string>
      </property>
      <property name="text">
       <string>Meseros</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAlmacenista">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	border: none;
	color: #A6A9A5;
}

:hover{
	color: #585858;
}
</string>
      </property>
      <property name="text">
       <string>Almacenista</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCocinero">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	border: none;
	color: #A6A9A5;
}

:hover{
	color: #585858;
}
</string>
      </property>
      <property name="text">
       <string>Cocinero</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lb_total">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>700</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Roboto&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>731</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Encabezados2">
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="elPatio">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(133, 133, 133);
</string>
      </property>
      <property name="text">
       <string>El Patio</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="line_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>360</y>
     <width>241</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 238, 238);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>651</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="MenuTop">
    <item>
     <layout class="QVBoxLayout" name="LetrasMenu">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="elPatio_2">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #585858
</string>
        </property>
        <property name="text">
         <string>Gesti√≥n de Usuarios</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="elPatio_3">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(133, 133, 133);
</string>
        </property>
        <property name="text">
         <string>Administra los Usuarios en tu Restaurante</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="BtnsMenu">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCrearUsuario">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">*{
	color: white;
	border-radius: 10px;
	background-color: #46B04A;
	border: none;
	padding: 5px;
}

:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}</string>
        </property>
        <property name="text">
         <string>Crear Usuario</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnEditarUsuario">
   <property name="geometry">
    <rect>
     <x>759</x>
     <y>700</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: white;
	border-radius: 10px;
	background-color: #46B04A;
	border: none;
	padding: 5px;
}

:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}</string>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>691</width>
     <height>511</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollareaalbum">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>689</width>
      <height>509</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridUsuarios">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="line_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>651</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 238, 238);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Recursos.qrc">:/iconos/resources/imgPersona2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nombreUsuario">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>310</y>
     <width>200</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #585858
</string>
   </property>
   <property name="text">
    <string>Nombre de Usuario</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tipoUsuario">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>380</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(133, 133, 133);
</string>
   </property>
   <property name="text">
    <string>Puesto: Mesero</string>
   </property>
  </widget>
  <widget class="QLabel" name="claveUsuario">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>430</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(133, 133, 133);
</string>
   </property>
   <property name="text">
    <string>Clave Mesero: 1654</string>
   </property>
  </widget>
  <zorder>sideBar</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>line</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>lb_total</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>line_3</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>btnEditarUsuario</zorder>
  <zorder>scrollArea</zorder>
  <zorder>line_2</zorder>
  <zorder>label</zorder>
  <zorder>nombreUsuario</zorder>
  <zorder>tipoUsuario</zorder>
  <zorder>claveUsuario</zorder>
 </widget>
 <resources>
  <include location="../Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
